modules/web/js/ballerina/ast/statements/if-else-statement.js

```diff
@@ -156,6 +153,12 @@ class IfElseStatement extends Statement {
             });
         });
 
+        if(jsonNode.else_statement) {
+            const elseStatement = new ElseStatement();
```

Comment:
Yeah. Changed to use BAST factory in https://github.com/ballerinalang/composer/pull/1980